keps/sig-node/24-apparmor/README.md

```diff
@@ -472,7 +475,10 @@ will rollout across nodes.
 
 ###### What specific metrics should inform a rollback?
 
-  N/A - the feature is already enabled by default since Kubernetes v1.4.
+  Clusters upgrading while using beta AppArmor annotations will want to ensure 
+  that profiles on upgraded nodes are loaded at the Kubelet's specified path prefix.
+  Containers of Pods loading AppArmor profiles will fail to start if they attempt to 
```

Comment:
there's probably an existing metrics for "pods that don't run for reasons like this".  Can you locate which metric to watch for this increasing and reference it here?